umpf

```diff
@@ -928,7 +928,10 @@ parse() {
 		test -n "${content}" || bailout "${cmd} line without value"
 		if ${STABLE}; then
```

Comment:
Hmmm. Maybe `if ${STABLE} || [ "${namespace}" = "format_patch" ];` is cleaner code than misusing the `STABLE` variable for this purpose, even if it is only used in that single location for the format-patch use case.